api/libproject/devstart.rb

```diff
@@ -1452,12 +1463,11 @@ def deploy_gcs_artifacts(cmd_name, args)
     # This file must be readable by all AoU researchers and the Leonardo service
     # account (https://github.com/DataBiosphere/leonardo/issues/220). Just make
     # these public since these assets are public in GitHub anyways.
-    run_inline_or_log(op.opts.dry_run, %W{
-      gsutil acl ch -u AllUsers:R
-      gs://#{gcc.project}-cluster-resources/setup_notebook_cluster.sh
-      gs://#{gcc.project}-cluster-resources/playground-extension.js
-      gs://#{gcc.project}-cluster-resources/aou-snippets-menu.js
-    })
+    if auth_domain_group.end_with?('firecloud.org')
```

Comment:
What is the intent with this check?

Reply:
This is intended to not run anything until the group exists, because we need to have deployed to create the group, and this is part of the deploy process. This sorta avoids circular requirements. (There's a step in the doc to temporarily make it TODO until you can create the auth domain.)

Reply:
OK, that sounds fine. I still don't understand the specific string check though. How about:

```
if !auth_domain_group.nil? and !auth_domain_group.empty?
```